TypeScript/channelType.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * This plugin checks if a channel is the specified type
+ *
+ * @author @NeoYaBoi [<@762918086349029386>]
+ * @version 1.0.0
+ * @example
+ * ```ts
+ * import { channelType } from "../plugins/channelType";
+ * import { channelType } from "discord.js"

```

Comment:
```suggestion
 * import type { ChannelType } from "discord.js"

```

Reply:
Yeah, the reason I removed this was because I was getting an error
![image](https://user-images.githubusercontent.com/80315475/185522304-9799e173-2a64-45ae-9978-9e71aae7039f.png)

Reply:
in commandModule folder, you don't have to import type, u can just import { ChannelType } from 'discord.js'

Reply:
But Evo wants to change the import for the example, isnt the example meant to be a command using the plugin?

Reply:
And if we did import it just for the plugin we would still get this error:
![image](https://user-images.githubusercontent.com/80315475/185535052-782b67cb-4d5d-436c-bacf-4b2c0496e5f0.png)

Reply:
Would it be beneficial to create a variable to use for the value and a separate one for the type?

Reply:
Nope, it should be ` import { ChannelType } from "discord.js"` when in the command, as it is being used as an actual value!

Reply:
in addition, should be import { ChannelType } from discord.js in plugin file.

Reply:
Ive required ChannelType instead of importing it and I have used typeof for the parameter